core/src/value/convert/from.rs

```diff
@@ -16,6 +16,76 @@ use either::{Either, Left, Right};
 #[cfg(feature = "indexmap")]
 use indexmap::{IndexMap, IndexSet};
 
+#[cfg(feature = "chrono")]
+impl<'js> FromJs<'js> for chrono::DateTime<chrono::Utc> {
+    fn from_js(ctx: Ctx<'js>, value: Value<'js>) -> Result<chrono::DateTime<chrono::Utc>> {
+        use chrono::TimeZone;
+
+        let global = unsafe { crate::qjs::JS_GetGlobalObject(ctx.ctx) };
+        let date_constructor = unsafe {
+            crate::qjs::JS_GetPropertyStr(
+                ctx.ctx,
+                global,
+                std::ffi::CStr::from_bytes_with_nul(b"Date\0")
+                    .unwrap()
+                    .as_ptr(),
+            )
+        };
+        unsafe { crate::qjs::JS_FreeValue(ctx.ctx, global) };
+
+        let is_date =
+            unsafe { crate::qjs::JS_IsInstanceOf(ctx.ctx, value.value, date_constructor) > 0 };
```

Comment:
Use [`Object::is_instance_of`](https://docs.rs/rquickjs/latest/rquickjs/struct.Object.html#method.is_instance_of) for such checks.